### PR TITLE
4.1/generic chart after file moves 

### PR DESCRIPTION
**Pull request by edli-dot**
Same edits as before but I had to create a new branch to get rid of the .devspace.

Merged 3- Setup_react_frontend and 4/generic_chart in order to get a functional chart alongside frontend styling. The component framework has been laid out. Next steps would be to introduce additional state logic to manage all the charts on the ChartContainer.

An eventual goal would be to have the ability for the user to have some sort of chart-generating autonomy that would create these new instances of charts. Also need to implement the self-reloading of the charts. Time interval based on what kind of time duration the query is looking at.

### CODE REVIEW

**Comment by tylershelton** [comment]
Hey @edli-dot, does this PR supercede PR #9 ? Or do both need to get merged into dev one after the other?

**Comment by edli-dot** [comment]
There were still some additional merge conflicts w/ dev that probably resulted in having to do some command line merge. I just commit an additional change that has resolved this issue. So now at this current state of time, PR#9 has now been closed and the files within this PR should all be working

File: assets/styles.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+  // display: relative;
+  // justify-content: center;
+  display: inline-flex;
+  justify-content: space-evenly;
+
+  background-color: rgb(49, 49, 50);
+  //   opacity: 0.5;
+}
+
+// #monitor {
```
Comment (tylershelton) [inline assets/styles.scss]: should we clean some of this commented-out CSS, or are we keeping it around for a reason?

File: src/index.html [hunk 2]
```diff
@@ -3,10 +3,15 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Kubernautics</title>
+    <title>KUBERNAUTICS</title>
+    <!-- <link rel="stylesheet" type="text/css" href="styles.css"> -->
   </head>
 
   <body>
-    <div id="root"></div>
+    <div id="root">
+      <!-- <video autoplay muted loop id="myVideo">
```
Comment (tylershelton) [inline src/index.html]: Is this commented out video needed, or can we leave it out?

**Review by tylershelton — CHANGES_REQUESTED** [review]
Great! Just a couple small questions about whether a few commented-out sections need to be included for the future or not. Have a look.

**Comment by edli-dot** [comment]
You made a good insight with those uncommented codes. The ones in the index.html and styles.scss were just for testing and unneeded so I took them out. The uncommented code that might be useful later on for us to reference for handling chart state is now moved into MonitoringComponent.jsx.

**Review by tylershelton — APPROVED** [review]
Looking good!

**Comment by tylershelton** [comment]
Merged into dev.

**Pull request merged**